paper/paper.md

```diff
@@ -18,119 +18,121 @@ authors:
 
-The object `Semidiscretization` couples the systems of a simulation and also creates the corresponding neighborhood searches for each system. The resulting ODE problem is then fed into the time integrator and is solved by an appropriate numerical time integration scheme.
+The semidiscretization couples the systems of a simulation and also creates the corresponding neighborhood searches for each system. The resulting ordinary differential equation (ODE) problem is then fed into the time integrator and is solved by an appropriate numerical time integration scheme.
 
 ![Main building blocks of TrixiParticles.jl \label{fig:structure}](structure.png){width=100%}
 
-The code presented here can be found on [GitHub](https://github.com/trixi-framework/TrixiParticles.jl),  along with a detailed [manual](https://trixi-framework.github.io/TrixiParticles.jl/stable/) explaining how to use the package. Additionally, we provide tutorials explaining how to set up a simulation for fluids, structure mechanics or a fluid-structure interaction.
-A collection of predefined simulation files can be found in the `examples` directory.
+TrixiParticles.jl is open source and available under the MIT license at [GitHub](https://github.com/trixi-framework/TrixiParticles.jl),  along with detailed [documentation](https://trixi-framework.github.io/TrixiParticles.jl/stable/) explaining how to use it. Additionally, we provide tutorials explaining how to set up a simulation of fluid flows, structure mechanics, or fluid-structure interaction.
+A collection of simulation setups to get started with can be found in the examples directory.
 
-### Feature highlights
+## Feature highlights
 
 So far, the following feature highlights have been implemented:
 
 * *Fluid Systems*
-    + `WeaklyCmpressibleSPHSystem` (WCSPH): Standard SPH method originally developed by [@Monaghan:1977] to simulate astrophysics applications.
-    + `EntropicallyDampedSPHSystem` (EDAC): As opposed to the WCSPH scheme, which uses an equation of state, this scheme uses a pressure evolution equation to calculate the pressure which is derived by [@Clausen:2013] and adapted to SPH by [@Ramachandran:2019].
+    + Weakly compressible SPH (WCSPH): Standard SPH method originally developed by [@Monaghan:1977] to simulate astrophysics applications.
+    + Entropically damped artificial compressible (EDAC) SPH: As opposed to the WCSPH scheme, which uses an equation of state, this scheme uses a pressure evolution equation to calculate the pressure, which is derived by [@Clausen:2013] and adapted to SPH by [@Ramachandran:2019].
```

Comment:
```suggestion
    + Entropically damped artificially compressible (EDAC) SPH: As opposed to the WCSPH scheme, which uses an equation of state, this scheme uses a pressure evolution equation to calculate the pressure, which is derived by [@Clausen:2013] and adapted to SPH by [@Ramachandran:2019].
```

Reply:
Is the SPH artificial or the compressibility?
The original name is "entropically damped artificial compressibility", isn't it? So that makes the compressibility artificial, which makes it "artificially compressible".

Reply:
In general, the WCSPH is an artificial compressibility (AC) scheme. 
According to [Clausen 2013](http://dx.doi.org/10.1103/PhysRevE.87.013309):
"Traditional AC schemes rely on an artificial equation of state that gives the pressure as a function of the density, which is known to enforce isentropic behavior."

Reply:
So it's artificially compressible SPH. The compressibility is artificial. When compressibility becomes an adjective, artificial becomes an adverb. Artificial compressible SPH would mean that the SPH is artificial and not the compressibility.

Reply:
Okay, right! "compressible SPH" doesn't make sense.
But "Entropically damped artiﬁcial compressibility for SPH" does make sense.